CMakeLists.txt

```diff
@@ -9,12 +9,18 @@ set(LWIP_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 
 set (LWIP_DEFINITIONS LWIP_DEBUG=1)
 
-if (${CMAKE_SYSTEM_NAME} STREQUAL "Windows")
-  add_subdirectory(${LWIP_DIR}/contrib/ports/win32/example_app)
-elseif(${CMAKE_SYSTEM_NAME} STREQUAL "Linux" OR ${CMAKE_SYSTEM_NAME} STREQUAL "Darwin")
-  add_subdirectory(${LWIP_DIR}/contrib/ports/unix/example_app)
+OPTION(LWIP_OPTION_ENABLE_EXAMPLE_APPS "Build example apps on win32/unix/darwin" ON)
```

Comment:
should be `win32/linux/darwin` not `win32/unix/darwin`, since `unix` is used for both `Linux` and `Darwin`.

Reply:
@DmitriiKostrov fixed in latest commit.

Reply:
@DmitriiKostrov Is there anything else that needs done to get this in? Been sitting 2 weeks now with no action.

Reply:
Hi, all looks fine for me. But I'm not a maintainer and cannot fully aprove it. Sorry